[Master] Add and update STS content (#3681)
wso2/docs-is: en/docs/includes/running-an-sts-client.md

## Run a sample STS Client

The following sample demonstrates the steps to run a Security Token Service (STS) client. The STS issues a security token to authenticate a user for your client application using WS-Trust protocols.

!!! note "Before you begin"
    You need to [configure WS-Trust]({{base_path}}/guides/identity-federation/configure-ws-trust/) for WSO2 Identity Server.

> To run the STS client

Reviewer: ```suggestion
To run the STS client:
```

1. Clone the [Identity Server samples](https://github.com/wso2/samples-is) repository.

> 2. In the cloned directory, navigate to `SAMPLES-IS/sts/sts-client` directory to build the STS client.  

Reviewer: ```suggestion
2. Open a terminal, navigate to `SAMPLES-IS/sts/sts-client` in the cloned directory.  
```

    !!! info

>         The sts-client sample sends the username and password defined in the `SAMPLES-IS/sts/sts-client/src/main/resources/client.properties` file. WSO2 Identity Server authenticates if the user is in the system, and if the user is in the system, a token is sent to the requesting party, which is the sts-client application in this case.

Reviewer: ```suggestion
        The `sts-client` sample sends the username and password defined in the `SAMPLES-IS/sts/sts-client/src/main/resources/client.properties` file. WSO2 Identity Server authenticates if the user is in the system, and if the user is in the system, a token is sent to the requesting party, which is the sts-client application in this case.
```

>         The default username and password defined are that of the default super administrator, and you can change it accordingly.

Reviewer: ```suggestion
        The default username and password are that of the default super administrator, and you can change it accordingly.
```
